Frends.AzureEventHub.Receive/Frends.AzureEventHub.Receive/Definitions/Enums.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Frends.AzureEventHub.Receive.Definitions;
+
+/// <summary>
+/// Authentication methods.
+/// </summary>
+public enum AuthenticationMethod
+{
+#pragma warning disable CS1591 // self explanatory.
```

Comment:
These could be explained with the summary. E.g. SASToken

Reply:
Modified. Disabled CS1591 because those summaries don't add any value to the content.